catroid/src/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -80,7 +84,21 @@ protected void onCreate(Bundle savedInstanceState) {
 			@Override
 			public void onDownloadStart(String url, String userAgent, String contentDisposition, String mimetype,
 					long contentLength) {
-				DownloadUtil.getInstance().prepareDownloadAndStartIfPossible(WebViewActivity.this, url);
+				Log.e("TAG", "contentDisposition: " + contentDisposition + "   " + mimetype);
```

Comment:
`"TAG"` is not really helpful when debugging. It should either be `WebViewActivity.class.getSimpleName()` or a `private static final String TAG = WebViewActivity.class.getSimpleName();` that you use here.

Also it's not an error, so `Log.e` is unwarranted, you might be looking for `.d()` or `i()`.